onset_maproom/layout.py

```diff
@@ -217,27 +230,29 @@ def controls_layout():
             #        ],
             #    ),
             #),
-           # Block(
-           #     "Yearly Statistics",
-           #     dbc.Select(
-           #         id="yearly_stats_input",
-           #         value="mean",
-           #         bs_size="sm",
-           #         options=[
-           #             {"label": "Mean", "value": "mean"},
-           #             {"label": "Standard deviation", "value": "stddev"},
-           #             {"label": "Probability of exceedance", "value": "pe"},
-           #         ],
-           #     ),
-           #     dbc.Collapse(
-           #         Sentence(
-           #             Number("probExcThresh1", 30, min=0, max=999),
-           #             "days since Early Start, as",
-           #             Units("poeunits"),
-           #         ),
-           #         id="probability-collapse",
-           #     ),
-           # ),
+            Block(
+                "Ask the map:",
+                dbc.Select(
+                    id="yearly_stats_input",
+                    value="monit",
+                    # bs_size="sm",
+                    options=[
+                        {"label": "Has germinating rain occured?", "value": "monit"},
+                        {"label": "When to prepare for planting?", "value": "mean"},
+                        #as of now, xr.std doesn't know how to deal with NaT
+                        #{"label": "Climatological Standard deviation", "value": "stddev"},
+                        {"label": "How risky to plant...", "value": "pe"},
```

Comment:
I commented on that in the users feedback document so that they can comment. I am not a fan of "mean: when to prepare for planting." 'mean' says too much or too little. Having Maprooms speak the language of users (ie Human rather than maths) is popular when there is a clear audience identified who we know what language they speak. Then documentation in the app describes the maths definitions.
In Senegal it seems the audience is identified, in Ethiopia, not so much, that's why the question whether it applies in Ethiopia set up is a valid one. I assumed, to start with, that yes it does, but we'll see what Tufa/Amanda say